### PR TITLE
 404 error code

### PR DESCRIPTION
**Pull request by squishypandakay**
Test added to check for a 404 code - complete and working  

https://ten10solutions.atlassian.net/browse/AUT-17

### CODE REVIEW

File: src/test/java/HomePage.java
Comment (BeatrizBravo) [inline src/test/java/HomePage.java:1]: good 404 request.

**Review by BeatrizBravo — APPROVED** [review]
The 404 error for the endpoint is accepted.

File: src/test/java/HomePage.java [hunk 1]
```diff
@@ -39,4 +39,24 @@ public void testRestAssured() {
...
+    @Test
+    public void testStatusCode(){
+//1.SetthebaseURI
+        RestAssured.baseURI="http://3.11.77.136";
+
+//2.SendanHTTPGETrequestto/index.phpandextracttheresponse
+        Response response=
+                given()
+                        .when()
+                        .get("/jfjufhdgb%20gvdm")
```
Comment (sauliuskucevicius) [inline src/test/java/HomePage.java:52]: Should this line have that random string contents in it?

Comment (sauliuskucevicius) [reply src/test/java/HomePage.java:52]: Aaah is it to force the 404 error?

**Review by sauliuskucevicius — APPROVED** [review]

**Pull request merged**